src/benchmark_runner.h

```diff
@@ -82,7 +83,6 @@ class BenchmarkRunner {
   // So only the first repetition has to find/calculate it,
   // the other repetitions will just use that precomputed iteration count.
 
-  PerfCountersMeasurement perf_counters_measurement;
   PerfCountersMeasurement* const perf_counters_measurement_ptr;
```

Comment:
nit: this no longer needs the `_ptr` suffix as it's the only way we store a reference to it.